share/commands/agent-check

```diff
@@ -259,6 +259,7 @@ check_and_fix_inputs() {
   then
     rm -rf ${CFE_DIR}/state/ncf-exclude-cache-*
     rm -f ${CFE_DIR}/inputs/rudder_promises_updated
+    ${CFE_BIN_DIR}/cf-agent -K -f failsafe.cf
```

Comment:
Why not use rudder agent update ?

Reply:
To match the other calls to CFEngine in this file, we can change them to use the Rudder command, but I would rather do it in 4.3.

Reply:
ok